components/datetime/src/format.rs

```diff
@@ -230,6 +231,18 @@ where
             ) as isize,
             field.length,
         )?,
+        FieldSymbol::Second(Second::FractionalSecond) => format_number(
+            w,
+            usize::from(
+                date_time
+                    .date_time()
+                    .fraction()
+                    .map(|f| f.milliseconds())
+                    .ok_or(Error::MissingInputField)?,
+            ) as isize,
+            field.length,
+        )?,
+        FieldSymbol::Second(Second::Millisecond) => todo!(), // ? : I think this is in fact milliseconds in day?
```

Comment:
Indeed.  We should drop the `A` field (milliseconds in day), returning an error if the pattern contains that character, just like we do with other fields that aren't implemented yet. I think `todo!()` is not right here, because we probably don't want to panic.